### ISSUE TITLE
Texture rendering issue in Linux

### ISSUE THREAD

**Issue by alissonlauffer** [jebbyk/OSBES-minecraft-bedrock-edition-shader#130]
## Description

I've tried OSBES inside my Linux PC (Thanks to the minecraft-linux project) and it works almost perfectly, with some issues. One of them is this texture rendering glitch that creates a border in all blocks:
![image](https://user-images.githubusercontent.com/29029632/114281245-b3692280-9a13-11eb-9e6e-b3cfdbfe37d1.png)

### Additional info
 - Operating system: Arch Linux
 - Graphics library: Mesa3D v21.0.2
 - GPU: Intel HD Graphics 4400
 - Minecraft version: v1.16.210
 - Shaders version: latest commit from the develop branch

**Comment by jebbyk** [comment]
hmm looks like some kind of texture coordinates precision issue

**Comment by alissonlauffer** [comment]
> hmm looks like some kind of texture coordinates precision issue

Yes, and it's more noticeable with semi-transparent blocks.
![image](https://user-images.githubusercontent.com/29029632/114282925-583c2d80-9a1d-11eb-9da4-9a2298a2e300.png)

**Comment by alissonlauffer** [comment]
Hi, I've just discovered this is a known Minecraft bug for x86/x86_64 devices (https://bugs.mojang.com/browse/MCPE-121068) and unrelated to OSBES. But OSBES makes the issue more perceptible. Here's how it looks like without the shaders (I just discovered it when looking at the furnace texture, otherwise it is almost imperceptible if you don't pay attention):
![image](https://user-images.githubusercontent.com/29029632/114484992-88342e00-9be1-11eb-972b-8ecf307dafab.png)

I'm really sorry for opening this issue. Next issues I'll investigate more before opening.
